### PR TITLE
Bugfix/inconsistent messages

### PR DESCRIPTION
**Pull request by flaviusRev**

### CODE REVIEW

File: src/services/WorkflowHandler.ts [hunk 1]
```diff
@@ -59,21 +60,33 @@ function processSteps(inputData: InputData | AIMessage): { stepsArray: string[][
...
+      Logger.warn('This was the AIMessage:', results.content.toString());
       Logger.warn('Error:', error);
-      return "Error processing your results, please try again or contact support.";
+      return {
+        error: "Error processing your results, please try again or contact support."
+      };
     }
   } 
-  return results;
+  return results; // Assume results are already a JSON object if not an AIMessage
```
Comment (kikoncuo) [inline src/services/WorkflowHandler.ts:85]: Delete comment

File: src/services/WorkflowHandler.ts [hunk 2]
```diff
@@ -172,11 +185,11 @@ export function getSolveNode(solverModel: Runnable, outputHandler: Function) {
       const chain = chatPromptTemplate.pipe(solverModel);
 
       const responseResult = await chain.invoke({task: state.task, plan: plan, results: state.results});
-
+      
       const finalResponse = JSON.stringify(processResults(responseResult));
 
       outputHandler('result', finalResponse);
-      Logger.log('Final response:', finalResponse)
+      console.log('Final response:', finalResponse)
```
Comment (kikoncuo) [inline src/services/WorkflowHandler.ts:192]: Change to logger

File: src/services/WorkflowHandler.ts [hunk 3]
```diff
@@ -172,11 +185,11 @@ export function getSolveNode(solverModel: Runnable, outputHandler: Function) {
       const chain = chatPromptTemplate.pipe(solverModel);
 
       const responseResult = await chain.invoke({task: state.task, plan: plan, results: state.results});
-
+
```
Comment (kikoncuo) [inline src/services/WorkflowHandler.ts:188]: Delete space

File: src/services/WorkflowHandler.ts [hunk 4]
```diff
@@ -59,21 +60,33 @@ function processSteps(inputData: InputData | AIMessage): { stepsArray: string[][
   return { stepsArray, fullPlan };
 }
 
-
-function processResults(results: any | AIMessage): string {
+function processResults(results: any | AIMessage): any {
```
Comment (kikoncuo) [inline src/services/WorkflowHandler.ts:63]: Type this, create new type in types file:
interface ErrorResponse {
  error: string;
}
And update any to ErrorResponse | Record<string, unknown>

**Review by kikoncuo — CHANGES_REQUESTED** [review]

File: src/services/WorkflowHandler.ts (hunk 3 above)
Comment (kikoncuo) [reply src/services/WorkflowHandler.ts:188]: Was this added by prettier, if that's the case I'm wrong and we can keep it as is

Comment (flaviusRev) [reply src/services/WorkflowHandler.ts]: Yeah, that's prettier

**Review by kikoncuo — APPROVED** [review]

**Pull request merged**